controller/config.go

```diff
@@ -15,6 +16,7 @@ const (
 	DefaultLabelPrefix         string = "azure.tags"
 	DefaultTagPrefix           string = "node.labels"
 	DefaultResourceGroupFilter string = "none"
+	DefaultInterval            string = "5m" // interval between updates
```

Comment:
Needing a comment is sometimes an indication that the variable could have a more descriptive name.

Reply:
MinSyncPeriod maybe? Since SyncPeriod in the manager struct is the max period.